src/KilnUniV3.sol

```diff
@@ -101,7 +103,7 @@ contract KilnUniV3 is KilnBase, TwapProduct {
     }
 
     function _swap(uint256 amount) internal override returns (uint256 swapped) {
-        require(GemLike(sell).approve(uniV3Router, amount));
+        GemLike(sell).approve(uniV3Router, amount);
```

Comment:
Although the `require` has been removed, there is still a subtle effect of the ERC20 interface in KilnBase.sol specifying a boolean return value for `approve`--Solidity will add a check that a value _is returned_, so tokens that don't return `true` on a successful approval will cause a revert here. I don't know offhand if any common tokens violate the standard in this way, but given that we support a non-standard `transfer`, we might as well support a non-standard `approve` (there's no security risk AFAICT). As a bonus, it should save a bit of gas.

Reply:
Nice catch. Handled in [7833018](https://github.com/makerdao/dss-kiln/pull/12/commits/7833018857da8ad4bbf9816188f58916497f0aea)